plugin/pom.xml

```diff
@@ -3,39 +3,34 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>com.googlecode.gwtquery.plugins</groupId>
-        <artifactId>draggable</artifactId>
+        <groupId>com.arcbees.gquery</groupId>
```

Comment:
maybe the groupId could be `com.arcbees.gquery.plugins`, the artifact id of the parent `<pluginName>-parent` and the artifactId `<pluginName>`, ie `draggable-parent` and `draggable`?
We could change tooltip and elastic for consistency, although people using it would have to change it too